universal-login-relayer/src/integration/sql/services/FutureWalletStore.ts

```diff
@@ -10,4 +10,12 @@ export class FutureWalletStore {
       .into('future_wallets')
       .returning('contractAddress');
   }
+
+  getGasPriceInToken(contractAddress: string) {
```

Comment:
Rename gasPriceInEth

Reply:
Should return gasPrice instead of object or if method returns object, return whole entity and rename to get